h/static/scripts/group-forms/components/EditGroupMembersForm.tsx

```diff
@@ -75,12 +77,45 @@ export default function EditGroupMembersForm({
   };
 
   const memberText = pluralize(members?.length ?? 0, 'member', 'members');
+  const normalizedFilter = filter.toLowerCase();
+
+  const filteredMembers = useMemo(() => {
+    if (!normalizedFilter || !members) {
+      return members;
+    }
+    // nb. We can get away with lower-casing name and filter to do
+    // case-insensitive search because of the character set restrictions on
+    // usernames. This would be incorrect for Unicode text.
+    return members.filter(m =>
+      m.username.toLowerCase().includes(normalizedFilter),
+    );
+  }, [normalizedFilter, members]);
```

Comment:
Since `normalizedFilter` is used only inside the callback to calculate `filteredMembers`, maybe it's worth moving there and make `filter` the dependency instead of `normalizedFilter`.

```suggestion

  const filteredMembers = useMemo(() => {
    if (!filter || !members) {
      return members;
    }
    // nb. We can get away with lower-casing name and filter to do
    // case-insensitive search because of the character set restrictions on
    // usernames. This would be incorrect for Unicode text.
    const normalizedFilter = filter.toLowerCase();
    return members.filter(m =>
      m.username.toLowerCase().includes(normalizedFilter),
    );
  }, [filter, members]);
```